addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -309,6 +313,59 @@ var Wysiwyg = Widget.extend({
             mediaDialog.on('cancel', this, resolve);
         });
     },
+    changeSelection(commandId, color, $dropDownToToggle = undefined) {
+        if(color === "") {
+            this.editor.execCommand('un' + commandId); // todo : not ideal : use a map to get cammands from actions ?
+        } else {
+            if (this.colorPickers[commandId].colorNames.indexOf(color) !== -1) {
+                // todo : find a better way to detect and send css variable
+                color = "var(--" + color + ")";
+            }
+            this.editor.execCommand(commandId, {color: color});
+        }
+        if($dropDownToToggle !== undefined) {
+            $dropDownToToggle.find(".dropdown-toggle").dropdown("toggle");
+        }
+    },
+    initColorPicker(colorPickerCommandId, $dropdownNode) {
+        return new Promise(()=> {
+            if (!this.colorPickers[colorPickerCommandId]) {
+                // Init the colorPalete for this color picker Dropdown.
+                this.colorPickers[colorPickerCommandId] = new ColorPaletteWidget(this, {});
+
+                // The <form> prevent the dropdown to automaticaly close when user click inside.
+                const $form = $('<form >');
+                $form.on("submit", function(e) {
+                    e.stopImmediatePropagation();
+                    e.preventDefault();
+                });
+                this.colorPickers[colorPickerCommandId].appendTo($form);
+                $form.appendTo($dropdownNode.find('.dropdown-menu'));
+
+                // Events listeners to trigger color changes
+                this.colorPickers[colorPickerCommandId].on('custom_color_picked', this, (e) => {
+                    this.changeSelection(colorPickerCommandId, e.data.color);
+                });
+                this.colorPickers[colorPickerCommandId].on('color_picked', this, (e) => {
+                    this.changeSelection(colorPickerCommandId, e.data.color, $dropdownNode);
+                });
+            }
+        });
+    },
+    toggleTextColorPicker() {
+        return new Promise((resolve)=> {
+            const $textColorDropdown = $("jw-toolbar .jw-dropdown-textcolor");
```

Comment:
The widget method `this.$()` or `this.$el.find()` is more scoped than the global `$`. So I suggest:
`this.$("jw-toolbar .jw-dropdown-textcolor");`